speech/getting-started/speech_recognition.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>Could use a bit more explanation. Why short audio first? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>It is just a matter of order short-long</p>